### DISCUSSION TITLE
How to create a blog post that redirect to an HTML file?

### DISCUSSION THREAD

**Discussion by pinshuai** [alshedivat/al-folio#1824] [Q&A]
I have a presentation that is in HTML format. How to write a blog post that will automatically redirect to the HTML file in the repo?

**Comment by george-gca** [comment]
We have an example that redirects to a pdf file. Just use it redirecting to an html.

**Reply by pinshuai** [reply]
Thanks for the tip. I tried that example below:
```
---
layout: post
title: A Quick Intro to Note-taking using Markdown
date: 2023-10-06
description: Slides of a quick introduction to note-taking using Markdown and Obsidian
redirect: /assets/html/intro_to_markdown/intro_to_markdown.html
---
Redirecting to another page.
```

The HTML file is rendered correctly on local server, however, it gave me this error when trying to deploy on Github Page.

```
[NbConvertApp] Converting notebook /tmp/jekyll-jupyter-notebook20231015-1914-en2p02.ipynb to html
          Conflict: The following destination is shared by multiple files.
                    The written file may end up with unexpected contents.
                    /home/runner/work/hydroaggie.github.io/hydroaggie.github.io/_site/blog/tag/jupyter/index.html
                     - blog/tag/jupyter/index.html
                     - blog/tag/jupyter/index.html
```

I am confused because I did not use jupyter notebook nor tags in my HTML. You can find a copy of the [HTML file](https://github.com/hydroaggie/hydroaggie.github.io/tree/master/assets/html/intro_to_markdown) on my Github. The HTML file is a presentation generated from Obsidian.

**Reply by pinshuai** [reply]
Ok, I fixed that error message by making tag names "Jupyter" and "jupyter" the same. However, the Github action was still unable to deploy the website with the added HTML post. I could not figure out what the error was.
